### DISCUSSION TITLE
Is there any specific way for request an update for a package?

### DISCUSSION THREAD

**Discussion by SamaelAmbrona** [microsoft/winget-pkgs#83479] [Q&A]
Good morning.

Should I just open an incident in case a package, for which there is an available update on software page, is not yet available for update in Winget?

How does it work with the updates? Do you get notifications from the apps or from users? How long does it take normally to get an update?

Best regards

**Comment by mdanish-kh** [comment]
> Should I just open an incident in case a package, for which there is an available update on software page, is not yet available for update in Winget?

Yes, you may open an [issue](https://github.com/microsoft/winget-pkgs/issues/new/choose) if you see an outdated package. 

> How does it work with the updates? Do you get notifications from the apps or from users?

WinGet is a highly community maintained repo and the packages are usually updated by community contributors. There is some level of automation but currently a lot of packages are manually updated. 

> How long does it take normally to get an update?

You can read about the process in [FAQ.md](https://github.com/microsoft/winget-pkgs/blob/master/FAQ.md#how-long-do-packages-take-to-be-published)
